### PR TITLE
added relative path option to cli

### PR DESCRIPTION
**Pull request by pax0s**

### CODE REVIEW

**Comment by jiminys** [comment]
could the argument be made that all urls should be src relative by default, and only revert to base relative if the file can't be found? Maybe a file location strategy could be used (start with src relative, then root relative, then base etc )?

**Comment by pax0s** [comment]
Hi, thanks for the comment.
IMO, to make it src relative, we will have to guess it by moving up the processed javascript path until finding a matching asset.
For example, if we had this structure :
package.json
src
|__ scripts
|&nbsp;&nbsp;&nbsp;&nbsp;|__ app
|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|__ app.js
|__ assets
&nbsp;&nbsp;&nbsp;&nbsp;|__ html
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|__ app.html
We would test for templateUrl = assets/html/app.html in app.js the following paths
src/scripts/app/assets/html/app.html, then src/scripts/assets/html/app.html, then finally src/assets/html/app.html.
The root would be the directory containing the package.json.
Are you OK with this description so that I can add it to this pull request ?

**Comment by pax0s** [comment]
Hello, could you please provide an answer to my last comment ? To approve my approach or suggest another solution so that I can implement it. I need this merge request because I currently applied this implementation locally but when I work on another workstation, I have to copy it and it really is not handy.

File: README.md [hunk 1]
```diff
@@ -97,9 +97,9 @@ Help wanted for implementing:
 - [ ] Jade
 - [ ] Less
 - [ ] Scss/Sass
-- [ ] Relative file path
+- [x] ~~Relative file path~~
 - [ ] Gulp and Grunt plugins
-- [ ] ES5 or 6 output options
+- [] ES5 or 6 output options
```
Comment (pablodenadai) [inline README.md]: Minor fix required here. Markdown is broken.

**Comment by pax0s** [comment]
Hello could you publish a new version so that I can use the relative path feature ?
Many thx in advance

**Comment by pablodenadai** [comment]
Deployed as `0.2.0`. See https://www.npmjs.com/package/angular2-inline-template-style.

Thanks for contributing :-)

**Comment by pax0s** [comment]
Thanks to you sir

**Pull request merged**